pastis/e2e_simulators/webbpsf_imaging.py

```diff
@@ -166,6 +162,32 @@ def set_up_nircam():
 
     return nircam, ote
 
+def set_up_cgi():
+    """
+    Return a configured instance of the CGI simulator on RST.
+
+    Sets up the Lyot stop and filter from the configfile, turns of science instrument (SI) internal WFE
+    :return: Tuple of CGI instance
+    """
+    webbpsf.setup_logging('ERROR')
+
+    #Set actuators numbesr
+    mode_in = CONFIG_PASTIS.get('RST', 'mode')
+    nbactuator = int(CONFIG_PASTIS.get('RST', 'nb_subapertures'))
+    nbactuator_in = int(np.sqrt(nbactuator))
+    if nbactuator_in**2 != nbactuator:
+        log.error('nb_subapertures in config_pastis.ini is not a square!')
```

Comment:
A square of what? Please add clarification to the error message. Also, I how about you actually raise an error if thsi actually happens, because there is not point in continuing to run anything. You can do:
```py
if nbactuator_in**2 != nbactuator:
    error_msg = f"The number of subapertures from config_pastis.ini is {nbactuator}, which is not the square of the actuators per row (={nbactuator_in})!"
    log.error(error_msg)
    raise ValueError(error_msg)
```